plugins/source/datadog/codegen/recipies/base.go

```diff
@@ -0,0 +1,112 @@
+package recipies
+
+import (
+	"bytes"
+	"embed"
+	"fmt"
+	"go/format"
+	"os"
+	"path"
+	"runtime"
+	"strings"
+	"text/template"
+
+	"github.com/cloudquery/plugin-sdk/codegen"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/iancoleman/strcase"
```

Comment:
Let's use our `caser` library from plugin-sdk everywhere (I realize this was copy-pasted from somewhere else, but we'll get round to fixing those other places eventually too)